src/main/java/com/github/distributedtracing/ContinuationLocal.java

```diff
@@ -0,0 +1,170 @@
+package com.github.distributedtracing;
```

Comment:
Is this ultimately what we want? Might as well state your intent and go for

org.continuation

from the start